public/app/features/users/UsersActionBar.tsx

```diff
@@ -33,7 +33,7 @@ export class UsersActionBar extends PureComponent<Props> {
       { label: 'Users', value: 'users' },
       { label: `Pending Invites (${pendingInvitesCount})`, value: 'invites' },
     ];
-    const canAddToOrg = contextSrv.hasPermission(AccessControlAction.OrgUsersAdd);
+    const canAddToOrg = contextSrv.hasPermission(AccessControlAction.UsersCreate);
```

Comment:
Nit. You could leverage the hasAccess method here.
```suggestion
       const canAddToOrg = contextSrv.hasAccess(AccessControlAction.OrgUsersAdd, canInvite);
```